src/pdfs/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -16,25 +16,29 @@ export interface ParcelClients {
     address_county?: string;
     address_postcode?: string;
     delivery_instructions?: string;
-    index: number;
-    total: number;
 }
 
 const styles = StyleSheet.create({
     page: {
         display: "flex",
         flexDirection: "column",
-        fontSize: "11px",
+        fontSize: "11pt",
     },
     cardWrapper: { border: "1pt solid black", margin: "10px" },
     bold: { fontFamily: "Helvetica-Bold", textTransform: "uppercase" },
```

Comment:
Maybe call this heading or title or something if it does more than just make it bold!